docs/src/main/asciidoc/build-items.adoc

```diff
@@ -0,0 +1,842 @@
+| +++Marker build item indicating that JMS is enabled+++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/artemis-core/deployment/src/main/java/io/quarkus/artemis/core/deployment/ArtemisCoreConfiguredBuildItem.java[`io.quarkus.artemis.core.deployment.ArtemisCoreConfiguredBuildItem`, window="_blank"]
+| +++Marker build item indicating that Artemis Core is configured+++
+|===
+== Artemis JMS
+[%header,cols=2*]
+|===
+|Class Name |Description
+| https://github.com/quarkusio/quarkus/blob/master/extensions/artemis-jms/deployment/src/main/java/io/quarkus/artemis/jms/deployment/ArtemisJmsConfiguredBuildItem.java[`io.quarkus.artemis.jms.deployment.ArtemisJmsConfiguredBuildItem`, window="_blank"]
+| +++Marker build item indicating that Artemis JMS is configured+++
+|===
+== Cache
+[%header,cols=2*]
+|===
+|Class Name |Description
+| https://github.com/quarkusio/quarkus/blob/master/extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/AdditionalCacheNameBuildItem.java[`io.quarkus.cache.deployment.AdditionalCacheNameBuildItem`, window="_blank"]
+| +++Build item used to ensure that a cache of the specified name is created at runtime This is used in order to create caches when means other than the standard cache annotations are used+++
+|===
+== container-image
+[%header,cols=2*]
+|===
+|Class Name |Description
+| https://github.com/quarkusio/quarkus/blob/master/extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageInfoBuildItem.java[`io.quarkus.container.spi.ContainerImageInfoBuildItem`, window="_blank"]
+| +++<i>No Javadoc found</i>+++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageLabelBuildItem.java[`io.quarkus.container.spi.ContainerImageLabelBuildItem`, window="_blank"]
+| +++A label to be added to the built container image This will only have an effect if the extension building the container image supports adding custom labels (like the Jib extension)+++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageBuildRequestBuildItem.java[`io.quarkus.container.spi.ContainerImageBuildRequestBuildItem`, window="_blank"]
+| +++<i>No Javadoc found</i>+++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/container-image/spi/src/main/java/io/quarkus/container/spi/BaseImageInfoBuildItem.java[`io.quarkus.container.spi.BaseImageInfoBuildItem`, window="_blank"]
+| +++<i>No Javadoc found</i>+++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImagePushRequestBuildItem.java[`io.quarkus.container.spi.ContainerImagePushRequestBuildItem`, window="_blank"]
+| +++<i>No Javadoc found</i>+++
+|===
+== core
```

Comment:
I guess the build items from core are the most important ones. Shouldn't we put them at the beginning?

Reply:
Done